seo-learning/seo-quiz.md

```diff
@@ -0,0 +1,205 @@
+Search Engine Optimization(SEO)
```

Comment:
Can you update main table in `README.md` file by including SEO quiz and providing link on it?